apps/maestro/src/features/ManageInterchainToken/ManageInterchainToken.tsx

```diff
@@ -149,24 +150,39 @@ export const ManageInterchainToken: FC<Props> = (props) => {
                   </li>
                 ))}
             </ul>
+            {options.some(
+              (o) => o.value === "mint" && o.isVisible(props) && o.disabled
+            ) && (
+              <p className="mt-1 text-sm text-error">
+                Minting is disabled because your Hedera account is not
+                associated with this token. Use the Association Status on the
+                card to associate, then try again.
+              </p>
+            )}
```

Comment:
Looks like we're adding something Hedera-related to a generic condition (minting is disabled).
Maybe we should also check the chain id?